numpyro/mcmc.py

```diff
@@ -103,7 +107,7 @@ def hmc(potential_fn, kinetic_fn=None, algo='NUTS'):
         euclidean kinetic energy.
     :param str algo: Whether to run ``HMC`` with fixed number of steps or ``NUTS``
         with adaptive path length. Default is ``NUTS``.
-    :return init_kernel, sample_kernel: Returns a tuple of callables, the first
+    :returns: a tuple of callables (`init_kernel`, `sample_kernel`): the first
```

Comment:
This will not get rendered correctly. You will need to enter the return types separately via `:rtype:`. Could you also ensure that we are using `:returns:` instead of `:return` or vice-versa? I think we have a mix of both.

Reply:
Thanks, I'll grep and make changes for `:return:` -> `:returns:` in this PR too.